sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Constants.cs

```diff
@@ -6,5 +6,6 @@ namespace Microsoft.Azure.WebJobs.Extensions.Storage.Common
     internal static class Constants
     {
         public const string DateTimeFormatString = "yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fffK";
+        public const int MaxDepthDefault = 128;
```

Comment:
This is too high. The default was 64 and leading to StackOverflowExceptions.

Reply:
Done

Reply:
Still 128, which is too high. See my original comment.